cookietemple/create/github_support.py

```diff
@@ -220,7 +223,6 @@ def prompt_github_repo(dot_cookietemple: Optional[dict]) -> Tuple[bool, bool, bo
         private = cookietemple_questionary_or_dot_cookietemple(function='confirm',  # type: ignore
                                                                question='Do you want your repository to be private?',
                                                                default='No')
-
```

Comment:
I like it when we always have a blank clear line before returning something.
I actually always add them and you remove them haha.